README.md

```diff
@@ -0,0 +1,84 @@
+# CNAB to OCI
+
+The intent of CNAB to OCI is to provide a translation tool in between the two formats 
+
+## Getting Started
+
+To get and build the project:
+```sh
+$ go get github.com/docker/cnab-to-oci
+$ cd $GOPATH/src/github.com/docker/cnab-to-oci
+$ make
+```
+By now you should have a binary of the project in the `bin` folder. To run it, execute:
+```sh
+$ bin/cnab-to-oci --help
+```
+
+### Prerequisites
+
+```
+- Make
+- Golang 1.9
+- Git
+```
+
+### Installing
+
+For installing, make sure your `$GOPATH/bin` makes part of the `$PATH`
+
+```sh
+$ make install
+```
+
+This will build and install `cnab-to-oci` into `$GOPATH/bin`
+
+### Usage
+This section is reserved to the usage of `cnab-to-oci` as a translation (binary) tool
```

Comment:
Missing a final point.

Reply:
Maybe change this part with something like: `cnab-to-oci` binary is a demonstration tool to push and pull a bundle to a registry. It comes with 3 commands: `push`, `pull` and `fixup`. ?

Reply:
Good!